android/app/src/main/java/com/example/medcarapp/ManualControl.java

```diff
@@ -47,13 +46,17 @@ protected void onCreate(Bundle savedInstanceState) {
         joystick.setOnMoveListener(new JoystickView.OnMoveListener() {
             int previousAngle = IMPOSSIBLE_ANGLE_AND_SPEED;
             int previousSpeed = IMPOSSIBLE_ANGLE_AND_SPEED;
+            TextView angleIndicator = (TextView)findViewById(R.id.angleIndicator);
+            TextView speedIndicator = (TextView)findViewById(R.id.speedIndicator);
```

Comment:
I think it may be better to include these in the onCreate() method rather than within the joystick onMovelistener.

Reply:
That must have been a simple oversight on our end. I agree this would make it more consistent with rest of the project. Gonna change this 👍